block.go

```diff
@@ -0,0 +1,12 @@
+package authenticatedblockstore
+
+// This file implements block primitives
+
+// Size of a Block can be arbitrary.
+// Intended to be 4096 to fit in a 4KB page.
+// Can be configurable later.
+const BlockSize = 16
+
+type Block struct {
+	data [BlockSize]byte
```

Comment:
Is this necessary? Why not just use [BlockSize]byte?

Reply:
Right, I think that makes more sense, I have changed it.